src/components/dev-hub/searchbar/Searchbar.js

```diff
@@ -135,6 +137,15 @@ const Searchbar = ({ isExpanded, setIsExpanded }) => {
         }
         setIsFocused(true);
     }, [isFocused]);
+
+    // Focus when slash pressed
+    const onSlashKeyUp = useCallback(({ key, keyCode }) => {
+        (key === '/' || keyCode === SLASH_KEY) && onExpand();
+    }, [onExpand]);
```

Comment:
This is great when the searchbar is condensed by default. However on screen widths above 1200px the searchbar is open by default and we would also like for it to focus there as well. Can we also include focusing that element here for the full size?

Reply:
Thanks for comment. I have fixed it and also have deleted prop 'value' for ExpandedSearchbar because component doesn't expect it. Have some suggestion to check SearchTextInput. SearchTextInput has 'isSearching' prop for styled component (mobile version) but for current variant this prop always has false. Searchbar doesn't use isFocused for mobile ExpandedSearchbar so like result 'isSearching' always is false for SearchTextInput. We can ignore  if it's unnecessary or create a task for the future.

Reply:
Great, thanks for finding. I have created a low priority ticket for us to tackle in the future for it. We pulled this component from another team so it is likely we didn't remove everything we didn't need to clean up, thanks!